scopes/compilation/compiler/compiler.docs.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-description: Provides a single interface for Component compilation
+description: Provides a single interface for the compilation of Components of different types
```

Comment:
A unified interface for component compilation.

Reply:
done